migrations/versions/f95e673af427_.py

```diff
@@ -0,0 +1,44 @@
+"""empty message
+
+Revision ID: f95e673af427
+Revises: 8454ee309fcc
+Create Date: 2020-04-07 12:17:05.573094
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'f95e673af427'
+down_revision = '8454ee309fcc'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('languages',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('name', sa.String(), nullable=True),
+    sa.PrimaryKeyConstraint('id')
+    )
+    op.create_table('methods',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('name', sa.String(), nullable=True),
+    sa.Column('snippet', sa.Text(), nullable=True),
```

Comment:
This file might not be necessary to push up; it's a duplicate of the existing one, and only a side-effect of what we had to do on your particular machine with the drop-migrate-upgrade process. If you removed it from the PR everyone should be able to use the existing migration.

Reply:
Buuut if we take it out it'll probably cause your particular environment problems...thoughts?

Reply:
Yeah, I wasn't sure about that. Do you want me to delete it officially?

Reply:
Group decided to keep it in.